app/src/main/java/com/theopensourcefamily/chessclock/ClocksPresenter.kt

```diff
@@ -0,0 +1,24 @@
+package com.theopensourcefamily.chessclock
+
+import io.reactivex.disposables.CompositeDisposable
+
+class ClocksPresenter {
```

Comment:
We will use MVP pattern? Maybe it's MVVM pattern, but I never worked with MVVM before. Anyway we can talk about what pattern should we use.

Reply:
I was thinking on going fully reactive with MVI pattern. This is kind of an experiment to me also, so let's see what happens! :D!

Reply:
Let's play! 😋